content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -113,6 +113,14 @@ <h3>Create a new Service</h3>
 				<code><b>echo "NODE_PORT=$NODE_PORT"</b></code></p>
 				<p>Now we can test that the app is exposed outside of the cluster using <code>curl</code>, the IP address of the Node and the externally exposed port:</p>
 				<p><code><b>curl http://"$(minikube ip):$NODE_PORT"</b></code></p>
+        {{< note >}}<p>If Docker is the container runtime, minikube tunnel is needed. This is because Docker Desktop does not support ports. <br>
+				<p>In a separate terminal window, execute:<br>
+				<code><b>$ minikube service kubernetes-bootcamp --url</b></code></p>
+				<p>The output looks like this:
+				<pre><b>http://127.0.0.1:51082<br>!  Because you are using a Docker driver on darwin, the terminal needs to be open to run it.</b></pre></p>
```

Comment:
Is this exact output? I'd think "darwin" should be capitalized,  but if this is a direct copy of a command output then it's correct.

Reply:
Hi @nate-double-u , thank you for pointing out! This is a direct copy from the terminal output, so am keeping it